js/interface.js

```diff
@@ -1190,9 +1247,15 @@ $('.file-manager-wrapper')
     hideSideActions();
 
     var type = $searchType.val();
+
+    showSpinner(true);
+
     search(type, term)
       .catch(function () {
         alert('Error on search files');
+      })
+      .finally(function () {
```

Comment:
`.finally` is not supported in IE11 and it doesn't look like the polyfill we use supports it

Reply:
I'm going to verify if we can add a polyfill for it.

Reply:
We will be adding a polyfill for this. 👍 approved